lib/src/custom_long_press_gesture_recognizer.dart

```diff
@@ -0,0 +1,22 @@
+import 'package:flutter/gestures.dart';
+
+const Duration _kLongPressTimeout = Duration(milliseconds: 250);
```

Comment:
Let's stick constants below the class. No need to litter the top of this file with random values. 

Please don't use leading "k"s, those "k"s don't tell us anything useful.

Please document each global property.